CONTRIBUTING.md

```diff
@@ -31,6 +31,46 @@ Dev guides:
 - [MacOS Dev Guide]
 - [Windows Dev Guide]
 
+
+Forking Kustomize and Making a Pull Request (PR):
+
+
+Navigate to the Kustomize repository on [GitHub](https://github.com/kubernetes-sigs/kustomize)  
+ Click on the "Fork" button in the top-right corner to create a fork of the repository under your GitHub account.  
+ Clone your forked repository to your local machine using the following command: `git clone https://github.com/your-username/kustomize.git`  
+Create a new branch for your changes: `git checkout -b my-branch-name`  
+Make the necessary changes to the codebase, add new features, fix bugs, etc.  
+Commit your changes and push them to your forked repository:  
+`git add .`  
+`git commit -m "Description of your changes"`  
+`git push origin my-branch-name`  
+Visit the original Kustomize repository and click on the "Compare & pull request" button to create a PR with your changes.  
+
+
+
+Building Kustomize Locally:
+
+
+Ensure that you have Go installed on your system. Refer to the [Go](https://golang.org/doc/install) installation guide for your platform.  
```

Comment:
It would be nice to specify that we recommend to use the version supported in go mod.